src/eval_framework/shared/types.py

```diff
@@ -2,6 +2,7 @@
 from typing import Type, TypeVar, cast
 
 from pydantic import BaseModel, ConfigDict, Field
+from typing_extensions import Annotated
```

Comment:
Unless we plan on supporting Python < 3.9:
```suggestion
from typing import Annotated
```

Reply:
Should probably just enable the "UP" ruff rules in the repo